cmd/tfsec/main.go

```diff
@@ -83,6 +85,13 @@ var rootCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		debug.Log("Loading custom checks...")
+		err = custom.Load(dir)
+		if err != nil {
+			debug.Log("An error occurred loading custom checks. %s", err)
```

Comment:
Do you think we should spit out a warning here for non-debug mode? Would be good to shout if somebody's checks are silently failing cos they screwed up their config or something.